main.go

```diff
@@ -45,6 +47,10 @@ func main() {
 func main_() int {
 	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stdout}).Level(zerolog.InfoLevel)
 
+	if strings.Contains(os.Args[0], "serve") {
+		os.Args = slices.Insert(os.Args, 1, "serve")
+	}
```

Comment:
1. Please enhance the lines around the switch in [L53](https://github.com/puhitaku/tftp-now/pull/6/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261L53) and detect the desired functionality there. Please do not manipulate the args since editing the args by the callee (called command) is not what you interact with commands today.

2. `strings.Contains(os.Args[0], "serve)` will be `true` if any part of the path, including directory names, has "serve". Please extract only the filename and do if-else with that. [Here's a demo](https://go.dev/play/p/IVxmZTcpgqK) that shows what exactly `os.Args[0]` has, and how to extract the filename.

3. Please use more tftp-now-specific program name like "tftp-now-serve" instead of "serve" which is too generic.

Reply:
Thank you for intent to accept and review the code. 
I tried implementing two methods: one to include file name detection logic in the switch-case statement, and the other to detect it before the switch-case. Which approach do you think is more appropriate? Approach (1) keeps the decision logic inside the switch, while approach (2) determines the command beforehand and avoids modifying the switch structure itself.
Additionally, I made the help command work as described in the readme.

Reply:
By the way, both methods only change the default behavior and perform the traditional behavior if the argument exists. This is because the tftp-now-serve command can also act as a client, so there is no need to bother to maintain the same binary for the client under a different name.

Reply:
Option 2 looks better to me. Let's proceed with the current branch.